### PR TITLE
BZ-1783799: Added path based route doc

### PR DESCRIPTION
**Pull request by sagidlow**
Applies to 4.5+
BZ Link: https://bugzilla.redhat.com/show_bug.cgi?id=1783799
QE Approval Required: @zhaozhanqi 
SME: @openshift/openshift-team-network-edge 
Preview Link: https://deploy-preview-31152--osdocs.netlify.app/openshift-enterprise/latest/networking/routes/route-configuration.html#nw-path-based-routes_route-configuration

### CODE REVIEW

**Review by lbarbeevargas — COMMENTED** [review]
A few feedback points, including to break up longer sentences if possible. Otherwise LGTM!

File: modules/nw-path-based-routes.adoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
+// Module filename: nw-path-based-routes.adoc
```
[the remark this answers is not in the document]
Comment (sagidlow) [reply modules/nw-path-based-routes.adoc:1]: @lbarbeevargas I just copied the comments from a similar networking file. I figured to keep it the same.

File: modules/nw-path-based-routes.adoc [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  path: "/test" <1>
+  to:
+    kind: Service
+    name: service-name
+----
+<1> The path is the only added attribute for a path-based route.
+
+[NOTE]
+====
+Path-based routing is not available when using passthrough TLS, as the router does not terminate TLS in that case and cannot read the contents of the request.
```
Comment (sagidlow) [inline modules/nw-path-based-routes.adoc:43]: @lbarbeevargas should pass through be hyphenated? I've used it at previous employers as hyphenated.

Comment (lbarbeevargas) [reply modules/nw-path-based-routes.adoc:43]: I'm not very familiar with networking, but searching a bit, it looks like it is "passthrough" when talking about routes.

**Comment by sagidlow** [comment]
@zhaozhanqi - Pending your approval.

**Comment by zhaozhanqi** [comment]
hi, @sagidlow,  assign this to @lihongan who is the leader of network-edge team of QE
I'm focusing on SDN testing.  thanks.

**Comment by lihongan** [comment]
@sagidlow  Yes, for routes related feature you can ask Network Edge QE: @quarterpin and me to review. Thanks.
@quarterpin Could you please take a look? thanks,

**Comment by quarterpin** [comment]
/lgtm

The doc looks good to me, Thanks!

**Pull request merged**